test_unit.py

```diff
@@ -1209,6 +1209,36 @@ def test_HTLC(self):
         self.assertEqual(p, base64.b64decode(golden))
         self.assertEqual(s.get_address(), golden_addr)
 
+    def test_periodic_fee(self):
```

Comment:
nit: test name could maybe be `periodic_pay` rather than `periodic_fee`

Reply:
`periodic_payment` :)